src/localization/en-CA.json

```diff
@@ -222,5 +223,7 @@
   "reset-email": "Send password reset email",
   "viewing": "Viewing",
   "featured-products": "featured products",
-  "incorect-email": "Please enter valid email"
+  "incorect-email": "Please enter valid email",
+  "configurable-product-message": "{SKUCode} is a configurable product. Please visit the product page to add it to the cart",
```

Comment:
Please coordinate these messages with @petro97 as he's adding a few for Bulk order as well